packages/react-component-library/src/components/RangeSliderE/partials/StyledHandle.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components'
+import { selectors } from '@defencedigital/design-tokens'
+import { transparentize } from 'polished'
+
+import { RANGE_SLIDER_HANDLE_COLOR } from '../constants'
+import { ThresholdColor } from '../useThresholdColor'
+import { StyledValue } from './StyledValue'
+
+interface StyledHandleProps {
+  $isActive?: boolean
+  $thresholdColor?: ThresholdColor
+}
+
+const { color } = selectors
+
+export const StyledHandle = styled.div<StyledHandleProps>`
+  left: var(--left);
```

Comment:
Is this compatible with IE11?

Reply:
Hmm looks like it needs a polyfill - I will investigate because this technique gives a nice performance improvement when dragging the handles in some environments.

Reply:
Polyfill added to CRA template and Storybook